### PR TITLE
N33: Why I choose to build my game from scratch

### PR DESCRIPTION
**Pull request by HeavyRain266**
_Part of #1010_

### CODE REVIEW

File: content/news/033/index.md [hunk 1]
```diff
@@ -179,6 +179,18 @@ _Discussions:
...
+
+[@HeavyRain266] published an article [Why I choose to build my game from scratch][why-article],
+short story about implementation of the game "Forbidden Valley" from scratch in Rust.
+Author tries to point out, how much you can learn from building your dream game
+without help of any game engine.
+
+_Discussions: [r/rust_gamedev][why-article]_
+
+[@HeavyRain266]: https://github.com/HeavyRain266
+[why-article]: https;//reddit.com
```
Comment (ozkriff) [inline content/news/033/index.md:192]: bad link

Comment (HeavyRain266) [reply content/news/033/index.md:192]: Oh woops, I forgot to replace that placeholder.

Comment (HeavyRain266) [reply content/news/033/index.md]: Fixed.

**Review by 17cupsofcoffee — APPROVED** [review]
:+1:

**Review by ozkriff — APPROVED** [review]

**Pull request merged**